act-rules-format.bs

```diff
@@ -304,22 +304,14 @@ See [[#appendix-data-example]] on how this could be expressed using JSON-LD and
 Update Management {#quality-updates}
 ------------------------------------
 
-### Version Numbers ### {#quality-updates-version}
-
-Each ACT Rule MUST have it's own version number. The version number has to follow the semantic versioning schema. Using the X.Y.Z schema in the following way:
-
-: X / Major updates:
-:: The major version number must be increased if the change can lead to new failure results.
+### Change Log ### {#quality-updates-changes}
 
-: Y / Minor updates:
-:: The minor version number must be increased if the test logic has been updated, which could lead to a a different result.
+It is important to keep track of what has changed in ACT rules, so that users of them can work out if changes in accessibility results are due to changes in the rules used in testing, rather than changes in the content itself. All changes to an ACT Rule that can change the outcome of a test MUST be recorded in a change log. The change log can either be part of the rule document itself, or be referenced from it.
 
-: Z / Patch updates:
-:: The patch version number must be increased if the change does not affect the result of a rule. This includes editorial changes, new issues on the issues list, an updated description, etc.
+Each new release of an ACT Rule MUST be identifiable with either a date or a version number. Additionally, a reference to the previous version of that rule MUST be available. It is recommended that for huge changes, a new rule is created and the old rule is deprecated. 
 
-### Change Log ### {#quality-updates-changes}
+An example of when a new rule should be created would be when going from a rule that tests the use of a blink element, and changes it to a rule that looks for animated style changes. This potentially adds lots of new issues that were previously out of scope. But for that same rule, adding a step to check if the blink element is positioned off screen, should be done by updating the existing rule.
```

Comment:
The problem with this example is that you wouldn't want to deprecate the original rule, as it would still be a valid test for the blink element.  You would, however want to create a new rule for the animated element checking - that part makes sense.